### ISSUE TITLE
[TUTORIAL] Backup Mnemonic / Seed Phrase 

### ISSUE THREAD

**Issue by LoicPandul** [DecouvreBitcoin/sovereign-university-data#716]
We'll need to think about tutorial topics for the wallet-backup subsection. I think it's a medium priority. We need to decide whether to create just a general tutorial explaining how to properly manage a mnemonic phrase or go for separate tutorials on the different solutions. But that might get redundant, and we'll need a lot of equipment. ^^

**Comment by BitPreto** [comment]
Good point, I put documentation label and tutorial proposal status to isolate this from other defined-content related issues

**Comment by LoicPandul** [comment]
Done #1230
